### PR TITLE
Removed BibTeX file type and included HTML and Markdown types

### PR DESCRIPTION
**Pull request by Simone-Thai**
Fixes [koppor/#547](https://github.com/koppor/jabref/issues/547?fbclid=IwAR2TtODoezs2F35XsIifHrrVOXfKlSTWcAQACgjDp5JJQGbfcJXjnZxSBEU) by removing the BibTeX dropdown option. I have also added HTML and Markdown filtering options as they are commonly used standard files.    

Screen Shot 2022-10-28 at 4 05 13 pm

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Simone-Thai** [comment]
I accidentally created a pull request in my own repository. Here is the link to it:
https://github.com/Simone-Thai/jabref/pull/2

**Comment by Simone-Thai** [comment]
I've also added a basic file icon to the dropdown.
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where a message about changed metadata would occur on saving although nothing changed. [#9159](https://github.com/JabRef/jabref/issues/9159)
 - When adding or editing a subgroup it is placed w.r.t. to alphabetical ordering rather than at the end. [koppor#577](https://github.com/koppor/jabref/issues/577)
 - We modified the Directory of Open Access Books (DOAB) fetcher so that it will now also fetch the ISBN when possible. [#8708](https://github.com/JabRef/jabref/issues/8708)
+- We modified the "Delete file" dialog and add the full file path to the dialog text. The file path in the title was changed to file name only. [koppor#534](https://github.com/koppor/jabref/issues/534)
+- Download from URL now automatically fills with URL from clipboard. [koppor#535](https://github.com/koppor/jabref/issues/535)
+- We added HTML and Markdown files to Find Unlinked Files and removed BibTex. Markdown was added as a StandardFileType and StandardExternalFileType [koppor#547](https://github.com/koppor/jabref/issues/547)
```
Comment (Siedlerchr) [inline CHANGELOG.md:50]:
```suggestion
- We added HTML and Markdown files to Find Unlinked Files and removed BibTex. [koppor#547](https://github.com/koppor/jabref/issues/547)
```
The changelog is targeted to end users, so implementation details are not necessary here

**Comment by Siedlerchr** [comment]
Please have a look at the failing unit and checkstyle tests

**Comment by Simone-Thai** [comment]
The Test `ExternalFileTypesTest` was failing because it didn't include markdown so I've added it in.

File: src/main/java/org/jabref/gui/externalfiletype/StandardExternalFileType.java [hunk 2]
```diff
@@ -28,8 +28,8 @@ public enum StandardExternalFileType implements ExternalFileType {
     TIFF(Localization.lang("%0 image", "TIFF"), "tiff", "image/tiff", "gimp", "picture", IconTheme.JabRefIcons.PICTURE),
     URL("URL", "html", "text/html", "firefox", "www", IconTheme.JabRefIcons.WWW),
     MHT("MHT", "mht", "multipart/related", "firefox", "www", IconTheme.JabRefIcons.WWW),
-    ePUB("ePUB", "epub", "application/epub+zip", "firefox", "www", IconTheme.JabRefIcons.WWW);
-
+    ePUB("ePUB", "epub", "application/epub+zip", "firefox", "www", IconTheme.JabRefIcons.WWW),
+    MARKDOWN("Markdown", "md", "application/x-latex", "emacs", "emacs", IconTheme.JabRefIcons.FILE_TEXT);
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/externalfiletype/StandardExternalFileType.java:32]: https://stackoverflow.com/a/25812177/3450689
The mime type needs to adjusted

**Review by Siedlerchr — CHANGES_REQUESTED** [review]

**Review by calixtus — APPROVED** [review]
Looks good to me

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**